tests/regression/test_fieldsplit_fieldsplit_aux_multigrid.py

```diff
@@ -10,6 +10,11 @@
 """
 import pytest
 from firedrake import *
+from firedrake.petsc import get_external_packages
+
+for package in ["mumps", "superlu_dist"]:
+    if package not in get_external_packages():
+        pytest.skip(f"{package} not installed with PETSc", allow_module_level=True)
```

Comment:
Why do we need both direct solvers for this file?
And why are the solver parameters in `test_fieldsplit_fieldsplit_aux_multigrid` hardcoded to mumps?

Reply:
I have no idea, I think we should ask the person who wrote the test. There's a lot going on here!

Reply:
This test works with either mumps for all direct solves, or with superlu_dist for all direct solves. It diverges with petsc direct solver and stagnates with pastix.

Instead of requiring both mumps and superlu_dist, I think we can relax this to checking whether either mumps or superlu_dist is the DEFAULT, and if not then skip.

Reply:
Also, to reduce the number of warnings the mesh comm should be passed to both the `VectorSpaceBasis` on lines 83 and 101 too.

Reply:
DONE

Reply:
I think this is now too lax. For me this test fails with either petsc or pastix as the direct solver, so there still needs to be a check that the DEFAULT_DIRECT_SOLVER is _one of_ mumps or superlu_dist. It just doesn't need to check that they are _both_ available.

Reply:
Okay, but the current Firedrake test suite doesn't pass without MUMPS anyway. PETSc `lu` only works in serial and pastix doesn't support all matrix types that MUMPS and superlu_dist do.

Reply:
https://github.com/firedrakeproject/firedrake/pull/3564

Reply:
Ok, if that's the support at the moment then this is fine.